TILApp/Views/CalendarView/CalendarViewController.swift

```diff
@@ -177,20 +84,16 @@ class CalendarViewController: UIViewController {
         $0.calendarWeekdayView.weekdayLabels[4].font = .boldSystemFont(ofSize: 20)
         $0.calendarWeekdayView.weekdayLabels[0].textColor = .darkGray
         $0.calendarWeekdayView.weekdayLabels[0].font = .boldSystemFont(ofSize: 20)
-        // 일요일 라벨의 textColor를 red로 설정
-        $0.calendarWeekdayView.weekdayLabels[6].textColor = .darkGray
-        $0.calendarWeekdayView.weekdayLabels[6].font = .boldSystemFont(ofSize: 20)
-
-        // 토요일 라벨 파란색으로
+        // 토요일 라벨
         $0.calendarWeekdayView.weekdayLabels[5].textColor = .darkGray
         $0.calendarWeekdayView.weekdayLabels[5].font = .boldSystemFont(ofSize: 20)
+        // 일요일 라벨
+        $0.calendarWeekdayView.weekdayLabels[6].textColor = .darkGray
+        $0.calendarWeekdayView.weekdayLabels[6].font = .boldSystemFont(ofSize: 20)
     }
 
     override func viewWillAppear(_ animated: Bool) {
```

Comment:
```suggestion
    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        navigationController?.isNavigatioinBarHidden = true
```

isNavigatioinBarHidden은 true로 설정해야 네비게이션바가 사라집니다.